bin/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+app="improtresk-api"
+
+DIR_ROOT="$(dirname "$(readlink -f "$0")")"
+
+cd $DIR_ROOT
+
+source ../../bin/activate
+
+python ../manage.py migrate
```

Comment:
Well, database from production will be cloned into staging database. Every build from master is deployed to staging, so I think it's safe to assume that all migrations will be tested against -the- database.

Created #79 for database backup. We don't need it for initial release. We can also add automatic rollback when the deploy fails. Not a big problem.